src/scrabble.js

```diff
@@ -1,6 +1,89 @@
+
+const lettersScore = {
+  'a': 1,
+  'b': 3,
+  'c': 3,
+  'd': 2,
+  'e': 1,
+  'f': 4,
+  'g': 2,
+  'h': 4,
+  'i': 1,
+  'j': 8,
+  'k': 5,
+  'l': 1,
+  'm': 3,
+  'n': 1,
+  'o': 1,
+  'p': 3,
+  'q': 10,
+  'r': 1,
+  's': 1,
+  't': 1,
+  'u': 1,
+  'v': 4,
+  'w': 4,
+  'x': 8,
+  'y': 4,
+  'z': 10
+}
+
+// Declare the class:
 class Scrabble {
   // Write your implementation here
+  constructor (letters) {
+    this.letters = letters
+  }
+
+  score () {
+    // let total to 0
+    let totalScore = 0
+    if (this.letters === null) {
+      return totalScore
+    }
+    const letters = this.letters.toLowerCase() 
 
+    // Do a for loop for each character of this.letters
+    for (let i = 0; i < letters.length; ++i) {
+      // add value of each letter to total
+      totalScore += lettersScore[letters.charAt(i)] || 0;
```

Comment:
Nice and straightforward solution.

Reply:
I agree 👌

Reply:
It's a simple and concise code which makes it easy to understand, especially for a beginner.